content/en/blog/2024/go-contrib-removal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Abandoned Go Contrib modules need code owners or will be removed
+linkTitle: Go Contrib modules ownership
+date: 2024-05-27
+author: >-
+  [Fabrizio Ferri-Benedetti](https://github.com/theletterf) (Splunk)
+issue: 4542
+sig: SIG Go
+# prettier-ignore
+cSpell:ignore: aws Benedetti Fabrizio Ferri gopkg labstack macaron moduled otelaws otelecho otellambda otelmacaron otelmongo otelmux
+---
+
+The Go SIG will remove Contrib modules that lack code owners. For each module,
+the removal is planned for no sooner than August 21, 2024, unless a code owner
+is found. The list includes the following modules:
+
+- `go.opentelemetry.io/otel/detectors/aws/ec2`
```

Comment:
Can we add hyperlinks to https://pkg.go.dev? Example:

```suggestion
- [`go.opentelemetry.io/otel/detectors/aws/ec2`](https://pkg.go.dev/go.opentelemetry.io/otel/detectors/aws/ec2)
```

Same comment applies for other modules

Reply:
Applying now!

Reply:
@pellared @dmathieu The first four modules seem to be hosted on contrib, too, not in the code SDK. Could you confirm?

Reply:
You are right.